Zend/zend_enum.c

```diff
@@ -44,7 +44,10 @@ zend_object *zend_enum_new(zval *result, zend_class_entry *ce, zend_string *case
 
 	ZVAL_STR_COPY(OBJ_PROP_NUM(zobj, 0), case_name);
 	if (backing_value_zv != NULL) {
-		ZVAL_COPY(OBJ_PROP_NUM(zobj, 1), backing_value_zv);
+		zval *prop = OBJ_PROP_NUM(zobj, 1);
+
+		ZVAL_COPY(prop, backing_value_zv);
+		Z_PROP_FLAG_P(prop) = 0;
```

Comment:
Why is this necessary? `IS_PROP_REINITABLE` should not be set here, right? `ZVAL_COPY` doesn't copy `extra`.

Reply:
I'm not sure either, but the value of the `extra` field is `-1` right after the `ZVAL_COPY`, so the `0` assignment is needed, otherwise a few tests fail on my local env (e.g. `Zend/tests/enum/no-write-properties.phpt`)